spec/controllers/spotlight/resources_controller_spec.rb

```diff
@@ -53,8 +53,7 @@
     describe 'POST create' do
       let(:blacklight_solr) { double }
       it 'create a resource' do
-        allow_any_instance_of(Spotlight::Resource).to receive(:reindex)
-        expect(blacklight_solr).to receive(:commit)
+        expect_any_instance_of(Spotlight::Resource).to receive(:reindex)
```

Comment:
here again it's curious that POST create   is  a "reindex"  -- normally, I would think PATCH or PUTS  would be a reindex and  POST would be more of an "add to index"